backend/src/notes/notes.service.ts

```diff
@@ -279,6 +285,24 @@ export class NotesService {
     }
   }
 
+  async checkIfNoteIdIsUsed(noteId: string): Promise<boolean> {
+    // Check if we already have a note with this id
+    const note = await this.noteRepository
+      .createQueryBuilder()
+      .where('note.publicId = :noteId')
+      .orWhere('note.id = :noteId')
+      .setParameter('noteId', noteId)
+      .getOne();
+    if (note !== null) {
+      this.logger.debug(
+        `A note with the alias '${noteId}' already exists.`,
```

Comment:
This is misleading and seems to be an implementation error. The method does only check publicIds and internal ids(?) but not aliases.